lib/src/global_router.dart

```diff
@@ -103,8 +103,7 @@ class GlobalRouter implements Router {
       isInitialRoute: false,
       arguments: arguments,
     );
-    final screenRoute = routeEntry
-        .value(routeSettings)
+    final screenRoute = getScreen(routeSettings)
```

Comment:
Considering we already have the routeEntry, it's not better to use it instead to call getScreen again?

Reply:
I mainly changed it, because to actually wrap the new screen into a nested Nuvigator, calling `getScreen` was needed. We can refactor this in the future to avoid this confusion, makes sense?